routes/handlers.go

```diff
@@ -21,6 +21,18 @@ func Index(c *fiber.Ctx) error {
 	})
 }
 
+// Setting will render the setting html template.
+func Setting(c *fiber.Ctx) error {
```

Comment:
what is the purpose of settings?
do you have anything in mind that we can add here later?

Reply:
Yes. It could be a settings page for user preferences. Adding it in so we can expand on future features and allow users to turn them on/off.

e.g. user can choose compact / expand table layouts when viewing database tables